engine/src/configuration/configuration.cpp

```diff
@@ -1,7 +1,7 @@
 /*
  * configuration.cpp
  *
- * Copyright (C) 2021-2022 Daniel Horn, Roy Falk, ministerofinformation,
```

Comment:
This really should be an auto-generated file. That's 6759 lines of code. I salute you.

Reply:
It **was** auto-generated, if that wasn't clear. I used your JsonStruct tool, with a few tweaks, and some changes to template.h and template.cpp. I've opened a couple of PRs with your changes.

Your tool turned out to be very handy. I certainly would not have wanted to write all that code manually.